<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="SLOG">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SLOG_LABEL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setProb">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>setProblem</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>158</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>0:00</string>
         </property>
         <property name="buddy">
          <cstring>timeDial</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="timeDial"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>remaining time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="timeBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SStart">
       <property name="text">
        <string>ServerStart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SStop">
       <property name="text">
        <string>ServerStop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
